machine/machine.go

```diff
@@ -191,6 +194,45 @@ func getNodeIdFromCpuBus(cpuBusPath string, threadId int) (int, error) {
 	return nodeId, nil
 }
 
+/* Look for per-node hugepages info using node id */
```

Comment:
Can you switch to normal `//` comments here to keep the consistency?

Reply:
Agreed.

Reply:
It is intended for the consistency.
Author(of this PR) and I referred below functions in this file.
Those functions have similar usage(parsing info from the given path) and they have comment style like below.
```
/* Look for sysfs cpu path containing core_id */ 
/* Such as: sys/bus/cpu/devices/cpu0/topology/core_id */ 
```
Take look below functions.
https://github.com/google/cadvisor/blob/f5e7ddf27add8bf6b3ead954d5b09d895a8597d7/machine/machine.go#L135
https://github.com/google/cadvisor/blob/f5e7ddf27add8bf6b3ead954d5b09d895a8597d7/machine/machine.go#L159